astropy/io/fits/hdu/table.py

```diff
@@ -790,6 +785,30 @@ class BinTableHDU(_TableBaseHDU):
     _extension = 'BINTABLE'
     _ext_comment = 'binary table extension'
 
+    def __init__(self, data=None, header=None, name=None, uint=False):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
+        data : array, `FITS_rec` or `astropy.table.Table`
```

Comment:
it will be clickable, so anyone not knowing that ``Table`` with a capital belongs to ``astropy.table`` will have a chance to click for enlightenment. Others seeing this as a dosctring from the interpreter, still get the whole namespace.